model/kops_metadata.go

```diff
@@ -28,6 +28,7 @@ type KopsMetadata struct {
 	ChangeRequest        *KopsMetadataRequestedState `json:"ChangeRequest,omitempty"`
 	RotatorRequest       *RotatorMetadata            `json:"RotatorRequest,omitempty"`
 	Warnings             []string                    `json:"Warnings,omitempty"`
+	Networking           interface{}                 `json:"networking,omitempty"`
```

Comment:
```suggestion
	Networking           interface{}                 `json:"Networking,omitempty"`
```

Same here. I know that you went with an `interface` because the networking payload can differ in kops, but are we sure that we can't marshal it into a custom struct that just gets the name of the networking type used? It would be nice to specify a `string` value here if we can.

Reply:
Yes, we should be able to marshell it. i'll make the change and push the commit.

Reply:
As suggested, displaying networking option as a string value. i.e 
<img width="699" alt="Screen Shot 2021-03-29 at 9 58 35 am" src="https://user-images.githubusercontent.com/53712140/112771035-6d0bdf00-9075-11eb-819c-1347e9a95c1f.png">